### PR TITLE
Fix error when loading GTFS with a subdirectory

### PR DESCRIPTION
**Pull request by Wilhansen**
`registerFeed` calls `getCsvReader` which may add entries in `errorStorage`.
The code before may manipulate `errorStorage` before it is initialized, causing an exception to happen
on the above circumstance.

### CODE REVIEW

**Comment by Wilhansen** [comment]
By the way, this allows `datatools-server` to pull GTFS archives directly off Github repositories simply by supplying the zip archive link.

**Comment by Wilhansen** [comment]
Got it. Should I still create an issue for this?

File: src/main/java/com/conveyal/gtfs/loader/JdbcGtfsLoader.java [hunk 1]
```diff
@@ -164,6 +165,20 @@ public FeedLoadResult loadTables () {
         }
         return result;
     }
+    
+    private void createSchema (String tablePrefix) {
```
[the remark this answers is not in the document]
Comment (Wilhansen) [reply src/main/java/com/conveyal/gtfs/loader/JdbcGtfsLoader.java]: Sure! Also made it static (and have it accept `Connection`) to make sure it does not access other fields.

File: src/main/java/com/conveyal/gtfs/loader/JdbcGtfsLoader.java [hunk 2]
```diff
@@ -129,11 +129,12 @@ public FeedLoadResult loadTables () {
             this.tablePrefix = randomIdString();
             result.filename = gtfsFilePath;
             result.uniqueIdentifier = tablePrefix;
-            registerFeed(gtfsFile);
+            createSchema(tablePrefix);
```
[the remark this answers is not in the document]
Comment (Wilhansen) [reply src/main/java/com/conveyal/gtfs/loader/JdbcGtfsLoader.java]: Added a comment and placed the relevant lines in a block. Hope it's not an overkill.

**Comment by landonreed** [comment]
No need to create an issue at this point in my view. Thanks for the changes!

**Pull request merged**